app/views/tag/_tags_pop.html.erb

```diff
@@ -0,0 +1,26 @@
+<div class="main_div">
+	<p class="small_heading"> Follow related tags: </p>
+	<%= form_with remote: true, url: '/subscribe/multiple/tag/:same' do |f| %>
+		<% tags.each do |tag| %>
```

Comment:
Please fix indentation issues here also

Reply:
I am so sorry, tags_pop partial is of no use anymore, kindly delete it.